### PR TITLE
[#10481] Remove video tour on home page

### LINKED ISSUE #10481: Home page: Remove video tour
The video tour in the home page is severely outdated and there are no current plans to create a new one.

### PR DESCRIPTION
**Pull request by hhdqirui**
Fixes #10481

**Outline of Solution**
Remove `div` tag for video tour. Make the remaining button center.

Before:
image

After:
image

### CODE REVIEW

**Comment by moziliar** [comment]
Hi @hhdqirui , for ease of review, it is advised to attach screenshot to the PR, ideally before-after change.

**Comment by hhdqirui** [comment]
> Hi @hhdqirui , for ease of review, it is advised to attach screenshot to the PR, ideally before-after change.

Hi. Thank you for the advice! Screenshots are attached in the PR description.

**Comment by moziliar** [comment]
@damithc hi prof just checking if my intuition is ok 🤣 is it just me or the "centred" button is not so centred anymore with the peeled page at the bottom left?

**Comment by damithc** [comment]
> @damithc hi prof just checking if my intuition is ok 🤣 is it just me or the "centred" button is not so centred anymore with the peeled page at the bottom left?

Doesn't make a big difference to my eye though.

**Comment by moziliar** [comment]
@damithc thank you for your input.

I guess when you compare the above and below images, you could see that the button doesn't fall right in between the two buttons before the change. But if you are ok with that, so am I!

File: src/web/app/pages-static/index-page/index-page.component.scss [hunk 1]
```diff
@@ -5,3 +5,8 @@ img {
 .margin-bottom-10px {
   margin-bottom: 10px;
 }
+
+.center {
```
Comment (moziliar) [inline src/web/app/pages-static/index-page/index-page.component.scss:9]: The class name is too generic. You could rename it to something like `.button-center` as different html tags require different ways to center, e.g. `.item-center`.

**Review by moziliar — CHANGES_REQUESTED** [review]
Code LGTM. A minor naming nit.

**Comment by hhdqirui** [comment]
> @hhdqirui what do you think? I honestly think it could go a little left (is it my OCD acting now...)

Hi @moziliar, I have made some manual measurement. It is indeed at the center. Maybe the screenshots are a bit misleading.

**Review by moziliar — APPROVED** [review]
LGTM!

**Review by rrtheonlyone — APPROVED** [review]

**Pull request merged**